server/routes/notification.js

```diff
@@ -0,0 +1,22 @@
+const express = require("express");
+const router = express.Router();
+const { validateMongoId, validateNotificationDetails } = require("../validate");
+const protect = require("../middleware/auth");
+const {
+  createNotification,
+  markNotificationAsRead,
+  getAllNotifications,
+  getUnreadNotifications,
+} = require("../controllers/notification");
+
+router
+  .route("/create")
+  .post(validateNotificationDetails, createNotification);
+
+router.route("/check/:_id").post(validateMongoId, markNotificationAsRead);
+
+router.route("/all").get(protect, getAllNotifications);
+
+router.route("/unread").get(protect, getUnreadNotifications);
```

Comment:
I think we can omit the keyword for one of these, `get("/")` is usually the default route so we can remove it for whichever makes more sense to be default.